Directory.Packages.props

```diff
@@ -103,6 +103,7 @@
     <PackageVersion Include="Grpc.Tools" Version="2.65.0" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="KubernetesClient" Version="14.0.9" />
+    <PackageVersion Include="JsonPatch.Net" Version="3.1.1" />
```

Comment:
???????????????????????

Reply:
Look at where it is used in ApplicationExecutor